src/EditorFeatures/Core/AddUsings/IAutomaticallyAddMissingUsings.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
```

Comment:
Is there a good reason for this? I thought we'd want to annotate any new files

Reply:
This got caught up in me merging across nullable being enabled. I'll fix to not disable